### PR TITLE
fixes #308 - added tests for home page.

### LINKED ISSUE #308: Add tests for home page controller.
Add tests for the home page controller.

### PR DESCRIPTION
**Pull request by djmgit**
fixes #308 - added tests for home page.
    
Fixes issue #308, Added tests for home page using jasmine and protractor.
Updated README.md

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by djmgit** [comment]
test

**Comment by djmgit** [comment]
@Achint08 @kavithaenair @SKrPl @vibhcool open for review

**Comment by djmgit** [comment]
@singhpratyush travis is  no more showing errors. Not sure about the codacy issues. Please re-review.

File: tests/home-spec.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+// tests for home page
+
+describe("Testing Loklak apps home page", function() {
+  beforeEach(function() {
+    browser.get("http://127.0.0.1:5000/");
+  });
+
+  it("should have a title", function() {
+    expect(browser.getTitle()).toEqual("Loklak Apps");
```
Comment (singhpratyush) [inline tests/home-spec.js]: Why do we need such tests? Under what circumstances can this go wrong?

**Comment by djmgit** [comment]
@singhpratyush I have removed the title test. It was not required, I had put it to make sure my test code is working :) The rest of the tests checks JS logic and data loaded from external json and js files. The coverage is very less (the middle section has not been covered). I will be adding more tests in the subsequent PRs. Please re-review.

**Comment by djmgit** [comment]
@vibhcool I have fixed the coday issue related to ';' , I am not sure about the other issues, it says browser and element are undefined, however they are objects provided by protractor framework.

**Pull request merged**